criu/cr-restore.c

```diff
@@ -1589,7 +1589,7 @@ static void restore_pgid(void)
 static int mount_proc(void)
 {
 	int fd, ret;
-	char proc_mountpoint[] = "crtools-proc.XXXXXX";
+	char proc_mountpoint[] = "/tmp/crtools-proc.XXXXXX";
```

Comment:
BTW: you can try to replace this mkdir,mount,open,umount,rmdir by fsopen,fsmount.
https://github.com/torvalds/linux/blob/master/samples/vfs/test-fsmount.c